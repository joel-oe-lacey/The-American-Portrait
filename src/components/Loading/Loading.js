import React from 'react';
import './Loading.scss';
import { Link } from 'react-router-dom';

const Loading = () => {
    return (
        <section className="alert">
            <section className="alert-center">
                <h1>
                    "The land flourished because it was fed from so many sources--because it was nourished by so many cultures and traditions and peoples." 
                </h1>
                <h2>
                    Lyndon B. Johnson
                </h2>
                <h3>
                    Loading Data...
                </h3>
                <Link to="/">Return Home</Link>
            </section>
        </section>
    )
}

export default Loading;