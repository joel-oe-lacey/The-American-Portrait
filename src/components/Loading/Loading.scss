.alert {
    height: 100%;
    width: 100%;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    background-color: #202020;
    color: white;
}

.alert-center {
    height: 40%;
    width: 40%;
    display: flex;
    flex-direction: column;
    justify-content: space-around;
    text-align: center;

    h1 {
        font-size: 2em;
    }

    h2 {
        font-size: 1.5em;
    }

    h3 {
        font-style: italic;
    }
}