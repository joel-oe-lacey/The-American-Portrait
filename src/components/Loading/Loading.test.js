import React from 'react';
import Loading from './Loading';
import { shallow } from 'enzyme'

describe('Loading', () => {
    it('should match a snapshot as expected', () => {
        const wrapper = shallow(<Loading />)

        expect(wrapper).toMatchSnapshot()
    })
})
