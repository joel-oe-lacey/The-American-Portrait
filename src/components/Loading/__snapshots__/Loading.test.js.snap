// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match a snapshot as expected 1`] = `
<section
  className="alert"
>
  <section
    className="alert-center"
  >
    <h1>
      "The land flourished because it was fed from so many sources--because it was nourished by so many cultures and traditions and peoples."
    </h1>
    <h2>
      Lyndon B. Johnson
    </h2>
    <h3>
      Loading Data...
    </h3>
    <Link
      to="/"
    >
      Return Home
    </Link>
  </section>
</section>
`;
